spec/order_spec.rb

```diff
@@ -0,0 +1,83 @@
+require 'order'
+require 'menu_module'
+
+describe Order do
+
+  subject(:order) { described_class.new }
+
+  let(:confirmation) { double :confirmation }
+
+  full_menu = Menu::STARTERS.merge(Menu::MAINS).merge(Menu::DESSERTS).merge(Menu::SIDES)
```

Comment:
Metrics/LineLength: Line is too long. [88/80] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)